uberfire-security/uberfire-security-management/uberfire-security-management-wildfly/uberfire-security-management-wildfly10/src/test/resources/logback-test.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Ah, OK @romartin I thought you was just asking about the prolog... 

@manstis Step away from this PR.. move along... stop being a bother.